lineman/app/components/flash/flash.scss

```diff
@@ -1,29 +1,21 @@
 .flash-root {
-  position: absolute;
-  top: $navbar-height;
-  width: 100%;
+  bottom: 20px;
+  left: 20px;
+  min-width: 220px;
+  position: fixed;
+  z-index: 200;
 }
 
 .flash-container {
-  padding: 5px 0;
+  background-color: $primary-text-color;
```

Comment:
Hmm.. why is the background a text color?

Actually, a screenshot of this would be baller.